src/devServer/SocketServer.ts

```diff
@@ -1,5 +1,5 @@
 import { FuseBox } from "../core/FuseBox";
-import { Server } from "ws";
```

Comment:
I think that belongs to the different PR ;-)

Reply:
yep but i dont know how to skip this commit :(

Reply:
I will check your first PR soon, then you can rebase